ui/Screen/UserProfile/Projects.svelte

```diff
@@ -1,23 +1,27 @@
 <script lang="typescript">
-  import { push } from "svelte-spa-router";
+  import { push } from "ui/src/router";
```

Comment:
How do you feel about using qualified imports consistently?
```suggestion
  import * as router from "ui/src/router";
```
In the code `router.push()` make’s it more explicit what it is doing.